projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -4209,6 +4235,35 @@ public function render() {
 		$field_width         = $this->get_attribute( 'width' );
 		$class               = 'date' === $field_type ? 'jp-contact-form-date' : $this->get_attribute( 'class' );
 
+		if ( ! empty( $this->get_attribute( 'borderradius' ) ) || is_numeric( $this->get_attribute( 'borderradius' ) ) ) {
+			$this->field_styles = 'border-radius: ' . (int) $this->get_attribute( 'borderradius' ) . 'px;';
+		}
+		if ( ! empty( $this->get_attribute( 'borderwidth' ) ) || is_numeric( $this->get_attribute( 'borderwidth' ) ) ) {
+			$this->field_styles .= 'border-width: ' . (int) $this->get_attribute( 'borderwidth' ) . 'px;';
+		}
+		if ( ! empty( $this->get_attribute( 'lineheight' ) ) || is_numeric( $this->get_attribute( 'lineheight' ) ) ) {
+			$this->field_styles .= 'line-height: ' . (int) $this->get_attribute( 'lineheight' ) . ';';
+		}
```

Comment:
Shouldn't it be "and" in those scenarios? Maybe I'm missing something.

```suggestion
		if ( ! empty( $this->get_attribute( 'borderradius' ) ) && is_numeric( $this->get_attribute( 'borderradius' ) ) ) {
			$this->field_styles = 'border-radius: ' . (int) $this->get_attribute( 'borderradius' ) . 'px;';
		}
		if ( ! empty( $this->get_attribute( 'borderwidth' ) ) && is_numeric( $this->get_attribute( 'borderwidth' ) ) ) {
			$this->field_styles .= 'border-width: ' . (int) $this->get_attribute( 'borderwidth' ) . 'px;';
		}
		if ( ! empty( $this->get_attribute( 'lineheight' ) ) && is_numeric( $this->get_attribute( 'lineheight' ) ) ) {
			$this->field_styles .= 'line-height: ' . (int) $this->get_attribute( 'lineheight' ) . ';';
		}
```

Reply:
In fact, I think we can even narrow it down to just `is_numeric` looking at it now. 👍